collage/layout.py

```diff
@@ -0,0 +1,213 @@
+import tkinter as tk
+from tkinter.colorchooser import askcolor
+from matplotlib import font_manager
+import numpy as np
+from datetime import datetime
```

Comment:
When we are packing application, we have only python3.7, python3.8 builtin libraries. Please add additional libraries to requirements.txt, and write tests for components which use them.